frontend/src/app/components/network-graph/network-graph.component.ts

```diff
@@ -17,6 +18,10 @@ interface SimulationLink extends SimulationLinkDatum<SimulationNode>, Link {
   styleUrls: ['./network-graph.component.css']
 })
 export class NetworkGraphComponent implements OnInit, OnChanges {
+  private static HIGHLIGHT_COLOR = '#2E41FF';
```

Comment:
Can we put this in some CSS somewhere?

Reply:
Would it make a difference? These colors are only internal in the component for coloring the nodes, the are not used anywhere else. We still have only one place to change. 

Moving them to css would make the logic more complex. 
https://github.com/raiden-network/explorer/pull/62/files#diff-a66689adaf934d8c6025c371e5381f41R324

Currently the `fill` changes depending on the state of the node (normal,highlighted, or selected). If we move the colors from the component to css we will have to add a `class` `attribute` for highlighted or selected that have static color values, and remove the `class` use a fill for the programmatically generated colors for the nodes in any other case.